cluster/examples/kubernetes/ceph/cluster-on-local-pvc.yaml

```diff
@@ -0,0 +1,340 @@
+#################################################################################################################
+# Define the settings for the rook-ceph cluster with common settings for a production cluster on top of bare metal.
+# At least three nodes are required in this example. See the documentation for more details on storage settings available.
+# For example, to create the cluster:
```

Comment:
How about adding a comment at the top that tells the assumptions of the example? We expect three nodes, each with two available disks. The example will certainly need to be modified with their host and device names.